### ISSUE TITLE
Doing something wrong?

### ISSUE THREAD

**Issue by spuchinsky** [bwydoogh/obsidian-force-view-mode-of-note#40]
I have added the keys obsidianEditingMode (live) and obsidianUIMode (preview) to two notes that I am trying to set-up as dashboards, but when I click out to another note it keeps the settings of the dashboard. I tried using the plugin to assign the default settings to every other folder, but all notes still open with in the reading view when clicking out from one of the dashboards. Am I doing something wrong? It cannot be that I have to add the plugin's front matter to every single note in my system.

**Comment by bwydoogh** [comment]
Did you enable the setting "Ignore force view when not in frontmatter"?

**Comment by spuchinsky** [comment]
I did, but after posting the message I tried using the plugin in my test vault and it worked perfectly. Then I shut off any plugin in my regular vault that I thought might cause a conflict with positive results.  I’ve turned back on a handful with no issues; still not sure which one caused the conflict, but thank you for your help!Regards,SergeSent from my iPhoneOn Jun 5, 2024, at 12:53 AM, Benny Wydooghe ***@***.***> wrote:﻿
Did you enable the setting "Ignore force view when not in frontmatter"?
